src/Catalog/Persistence/AzureStorage.cs

```diff
@@ -15,14 +16,24 @@ namespace NuGet.Services.Metadata.Catalog.Persistence
 {
     public class AzureStorage : Storage
     {
-        private readonly CloudBlobDirectory _directory;
+        private const int _serverTimeoutSeconds = 30;
+        private const int _defaultMaxTimeoutInSeconds = 600;
```

Comment:
Nit could we pass a `TimeSpan` around instead of an int? In general we should use data types that represent the value as closely as possible. Leads to code that is easier to understand.